client/blocks/comment-detail/comment-detail-author.jsx

```diff
@@ -0,0 +1,112 @@
+/**
+ * External dependencies
+ */
+import React, { Component, PropTypes } from 'react';
+import Gridicon from 'gridicons';
+import { localize } from 'i18n-calypso';
+import classNames from 'classnames';
+
+export class CommentDetailAuthor extends Component {
+	static propTypes = {
+		author: PropTypes.object,
+		commentDate: PropTypes.string,
+	};
+
+	state = {
+		isExpanded: false,
+	};
+
+	toggleExpanded = () => {
+		this.setState( { isExpanded: ! this.state.isExpanded } );
+	}
+
+	render() {
+		const {
+			author,
+			commentDate,
+			moment,
+			translate,
+		} = this.props;
+		const { isExpanded } = this.state;
+
+		const classes = classNames( 'comment-detail__author', {
+			'is-expanded': isExpanded,
+		} );
+
+		return (
+			<div className={ classes }>
+				<div className="comment-detail__author-preview">
+					<div className="comment-detail__author-avatar">
+						<img src={ author.avatar_URL } />
```

Comment:
Right and we used to have a pattern where we passed though a full `site` object all over 😄 . 
 
User is a much smaller object, so I'd prefer if we used flat props, but I wouldn't block this PR. (That 🐍  case though. Hah.)

On that note, as a heads up `users` hasn't been fully ported to the redux store. We currently are only guaranteed that the current-user is available:  https://github.com/Automattic/wp-calypso/blob/master/client/boot/common.js#L179